### PR TITLE
btag@HLT paths modification to be consistent with the new HLT menu

### PR DESCRIPTION
**Pull request by natalia-korneeva**
Minor modification of names of HLT paths to monitor.

### CODE REVIEW

File: HLTriggerOffline/Btag/python/HltBtagPostValidation_cff.py [hunk 1]
```diff
@@ -4,11 +4,11 @@
 #define HltBTagPostValidation for the b-tag DQM validation (efficiency and mistagrate plot)
 HltBTagPostValidation = DQMEDHarvester("HLTBTagHarvestingAnalyzer",
 	HLTPathNames = cms.vstring(
-	'HLT_PFMET120_',
-	'HLT_HT300PT30_QuadJet_75_60_45_40_v',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_',
```
Comment (mtosi) [inline HLTriggerOffline/Btag/python/HltBtagPostValidation_cff.py]: shouldn't be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`
?

File: HLTriggerOffline/Btag/python/HltBtagValidation_cff.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #denominator trigger
 hltBtagTriggerSelection = cms.EDFilter( "TriggerResultsFilter",
     triggerConditions = cms.vstring(
-      "HLT_PFMET120_* OR HLT_HT300PT30_QuadJet_75_60_45_40_v* OR HLT_PFHT380_SixJet32_DoubleBTagCSV_* OR HLT_Ele28_eta2p1_WPTight_Gsf_HT150_* OR HLT_IsoMu24_eta2p1_*"),
+      "HLT_PFMET120_PFMHT120_IDTight_v* OR HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v* OR HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_* OR HLT_Ele28_eta2p1_WPTight_Gsf_HT150_* OR HLT_IsoMu24_eta2p1_v*"),
```
Comment (mtosi) [inline HLTriggerOffline/Btag/python/HltBtagValidation_cff.py]: shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

File: HLTriggerOffline/Btag/python/HltBtagValidation_cff.py [hunk 3]
```diff
@@ -19,17 +19,17 @@
...
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_',
```
Comment (mtosi) [inline HLTriggerOffline/Btag/python/HltBtagValidation_cff.py]: shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

File: HLTriggerOffline/Btag/python/HltBtagValidation_cff.py [hunk 4]
```diff
@@ -43,11 +43,11 @@
 hltbTagValidation = cms.EDAnalyzer("HLTBTagPerformanceAnalyzer",
 	TriggerResults = cms.InputTag('TriggerResults','','HLT'),
 	HLTPathNames =cms.vstring(
-	'HLT_PFMET120_',
-	'HLT_HT300PT30_QuadJet_75_60_45_40_v',
-	'HLT_PFHT380_SixJet32_DoubleBTagCSV_',
+	'HLT_PFMET120_PFMHT120_IDTight_v',
+	'HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v',
+	'HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_',
```
Comment (mtosi) [inline HLTriggerOffline/Btag/python/HltBtagValidation_cff.py]: shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

File: HLTriggerOffline/Btag/test/config.ini [hunk 5]
```diff
@@ -8,42 +8,42 @@ files= root://cms-xrd-global.cern.ch//store/relval/CMSSW_8_1_0_pre7/RelValTTbar_
...
-HLT_PFMET120_:
+HLT_PFMET120_PFMHT120_IDTight_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
```
Comment (mtosi) [inline HLTriggerOffline/Btag/test/config.ini]: shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

File: HLTriggerOffline/Btag/test/config.ini [hunk 6]
```diff
@@ -8,42 +8,42 @@ files= root://cms-xrd-global.cern.ch//store/relval/CMSSW_8_1_0_pre7/RelValTTbar_
...
 	hltVerticesL3
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltFastPrimaryVertex
 	hltFastPVPixelVertices
 	hltVerticesL3
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
```
Comment (mtosi) [inline HLTriggerOffline/Btag/test/config.ini]: shouldn't it be
`HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_v`

File: HLTriggerOffline/Btag/test/config.ini [hunk 7]
```diff
@@ -8,42 +8,42 @@ files= root://cms-xrd-global.cern.ch//store/relval/CMSSW_8_1_0_pre7/RelValTTbar_
...
 
-HLT_HT300PT30_QuadJet_75_60_45_40_v:
+HLT_PFHT300PT30_QuadPFJet_75_60_45_40_v:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
-HLT_PFHT380_SixJet32_DoubleBTagCSV_:
+HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_:
 	hltCombinedSecondaryVertexBJetTagsCalo
 
 HLT_Ele28_eta2p1_WPTight_Gsf_HT150_:
```
Comment (mtosi) [inline HLTriggerOffline/Btag/test/config.ini]: are you sure about this ?
shouldn't it be
`  HLT_Ele28_eta2p1_WPTight_Gsf_HT150_v`

**Comment by mtosi** [comment]
@cmsbuild, please test

File: HLTriggerOffline/Btag/test/config.ini (hunk 6 above)
Comment (natalia-korneeva) [reply HLTriggerOffline/Btag/test/config.ini]: @mtosi in this case it should be HLT_PFHT380_SixPFJet32_DoublePFBTagCSV_2p2_v , but in the previous version it was HLT_PFHT380_SixJet32_DoubleBTagCSV_p075_v, so keeping in mind possible future changes I would leave this path as is (without 2p2_v)

File: HLTriggerOffline/Btag/test/config.ini (hunk 7 above)
Comment (natalia-korneeva) [reply HLTriggerOffline/Btag/test/config.ini]: Yes, there is only one HLT path that could be found with this pattern. To be more strict it is of course possible to add _v

**Comment by mtosi** [comment]
did you test the config ?
are you sure you are selecting events ?

  mia

**Comment by natalia-korneeva** [comment]
@mtosi yes, everything is fine, I've got all the histograms successfully filled.

**Comment by dmitrijus** [comment]
+1

**Comment by mtosi** [comment]
@cmsbuild, please test

**Comment by dmitrijus** [comment]
+1

File: HLTriggerOffline/Btag/python/HltBtagValidation_cff.py (hunk 3 above)
Comment (davidlange6) [reply HLTriggerOffline/Btag/python/HltBtagValidation_cff.py]: was this question resolved? @natalia-korneeva

Comment (natalia-korneeva) [reply HLTriggerOffline/Btag/python/HltBtagValidation_cff.py]: @davidlange6 yes as @mtosi agrees with my point and doesn't insist on finishing with _v

**Comment by davidlange6** [comment]
+1

**Pull request merged**